e2e/lib/k8up.bash

```diff
@@ -120,6 +120,19 @@ given_a_subject() {
 	echo "✅  The subject is ready"
 }
 
+given_an_annotated_subject() {
+	if [ "${#}" != 2 ]; then
+		errcho "$0 Expected 2 arguments, got ${#}."
+		exit 1
+	fi
```

Comment:
Not sure that if-clause is worth maintaining. We're writing tests here, not really a library. I would just assume that the tester supplies the required arguments by reading the code.